auth/authRouter.js

```diff
@@ -0,0 +1,56 @@
+const router = require("express").Router();
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+
+const db = require("../data/helpers/dbHelper.js");
+const secret =
+  require("../secrets/jwtSecret.js").jwtSecret || "not secure secret";
+
+router.post("/register", async (req, res) => {
+  const user = req.body;
+  try {
+    const hashword = bcrypt.hashSync(user.password, 12);
+    user.password = hashword;
+    const added = await db.registerUser(user);
+    if (added) {
+      res.status(201).json(added);
+    } else {
+      res.status(400).json({ error: "Username already exists" });
+    }
+  } catch (err) {
+    console.log(err);
+    res.status(500).json({ error: "There was a problem registering the user" });
+  }
+});
+
+router.post("/login", async (req, res) => {
+  const { username, password } = req.body;
+  try {
+    const checkUser = await db.findUserByName(username);
+    if (checkUser && bcrypt.compareSync(password, checkUser.password)) {
+      const token = createToken(checkUser);
+      res.status(200).json({ message: `Welcome ${username}`, token: token });
```

Comment:
Since the `token` variable has the same as its key, you could shorten it down:
```javascript
  res.status(200).json({ message: `Welcome ${username}`, token });
```